R/LD.R

```diff
@@ -241,6 +256,7 @@ load_LD_matrix <- function(LD_meta_file_path, region, extract_coordinates = NULL
 
     # Create a combined LD matrix
     combined_LD_matrix <- create_combined_LD_matrix(combined_LD_matrices, combined_LD_variants)
+    combined_LD_variants <- combined_LD_variants[match(rownames(combined_LD_matrix),combined_LD_variants$variants),]
```

Comment:
Here my undersatnding is that the difference is just one variant? then we should not match it like this. We should address it specifically and clearly.

Reply:
if you use my `mergeVectors` above, it should work and you can remove this line?